### DISCUSSION TITLE
Is findUnusedIssueHandlerSuppression option still alive?

### DISCUSSION THREAD

**Discussion by nevmerzhitsky** [vimeo/psalm#10703] [Q&A] [answered]
I see a mention of the option `findUnusedIssueHandlerSuppression` in [docs](https://psalm.dev/docs/running_psalm/configuration/#findunusedissuehandlersuppression), but [config.xsd](https://github.com/vimeo/psalm/blob/fe2c67ec89f358940f90db05efd2d663388b45a6/config.xsd) does not declare it. Also, I can't find it in the source code, as well the `UnusedIssueHandlerSuppression` class. Also, I see this option does not affect my project when I add some unused suppressions like:

```xml
<issueHandlers>
    <ImplicitToStringCast errorLevel="suppress"/>
    <UndefinedDocblockClass errorLevel="suppress"/>
    <MissingDependency errorLevel="suppress"/>
</issueHandlers>
```

So, is this option still available in psalm?

**Comment by weirdan** [comment, marked as answer]
It's a new option available in the `master` branch.
